script/collect.js

```diff
@@ -9,19 +9,20 @@ const {nodeVersions} = require('../package.json')
 collect()
 
 async function collect () {
-  for (const version of nodeVersions) {
-    await getDocsForNodeVersion(version).catch(err => {
+  for (const major in nodeVersions) {
+    const version = nodeVersions[major]
+    await getDocsForNodeVersion(major, version).catch(err => {
       console.error(`problem fetching version: ${version}`)
       console.error(err)
     })
   }
 }
 
-async function getDocsForNodeVersion (version) {
-  const docDir = path.join(__dirname, `../content/${version}/en-US/doc`)
-  const tempDir = path.join(__dirname, `../temp/${version}`)
+async function getDocsForNodeVersion (major, version) {
+  const docDir = path.join(__dirname, `../content/${major}/en-US/doc`)
```

Comment:
Case Study:
1. Fetch(v10.1.0) :arrow_right: creates a docDir with label `v10.x` which is the *MAJOR* version
2. Fetch(v10.2.0) :arrow_right: already finds a docDir with label `v10.x` which shall log `docs for 10.2.0 have already been collected; skipping` and skip fetching [ref: `collect.js:26`

Reply:
Yanked that out! Thanks for noticing, @maddhruv